pkg/spec/spec.go

```diff
@@ -32,7 +32,8 @@ type Function struct {
 // FunctionSpec contains func specification
 type FunctionSpec struct {
 	Handler  string             `json:"handler"`
-	Function string             `json:"function"`
+	File     string             `json:"file"`
+	Checksum string             `json:"checksum"`
```

Comment:
Including a checksum is a good idea.  We should probably make the checksum include a prefix saying which algorithm was used, even if we only support one algorithm for now.  (eg: "sha256:xxxx")